packages/app/src/webapp/views/sidecar.ts

```diff
@@ -1002,7 +1002,7 @@ const setVisible = (sidecar: Sidecar) => {
 
   scrollIntoView()
 
-  setTimeout(() => eventBus.emit('/sidecar/toggle', { sidecar, tab }), 600)
+  eventBus.emit('/sidecar/toggle', { sidecar, tab })
```

Comment:
we need the setTimeout, but the timeout value can probably be zero

Reply:
updated thanks!